orchestra/migrations/0018_auto_20151007_1451.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+def update_projects(apps, schema_editor):
+    Project = apps.get_model('orchestra', 'Project')
+    WorkflowVersion = apps.get_model('orchestra', 'WorkflowVersion')
+
+    for project in Project.objects.all():
+        project.workflow_version = WorkflowVersion.objects.get(
+            slug=project.workflow_slug)
```

Comment:
This assumes that when an existing workflow is implemented with a json manifest, the version_slug matches the old workflow_slug.